web/src/main/java/com/es/phoneshop/web/controller/pages/ProductListPageController.java

```diff
@@ -40,6 +40,7 @@ public String showProductList(@RequestParam(value = "sortingParameter", defaultV
         model.addAttribute("finalPage", productListAttributes.getFinalPage());
         model.addAttribute("currentPage", productListAttributes.getCurrentPage());
         model.addAttribute("cartTotal", cartService.getCart().getTotal());
+        model.addAttribute("count", cartService.countProducts());
```

Comment:
use ControllerAdvice in order to avoid setting 'count' model attribute in multiple controllers